gatsby/content/projects/2020-01-04-native-gitter-bridge.mdx

```diff
@@ -0,0 +1,37 @@
+---
+layout: project
+title: Native Gitter bridge
+categories:
+  - bridge
+description: Bridges Gitter to the Matrix network
+author: Gitter
+maturity: Released
+language: JavaScript
+license: Apache-2.0
+repo: https://gitlab.com/gitterHQ/webapp
+thumbnail: /docs/projects/images/gitter-logo.svg
+bridges: Gitter
+featured: true
+---
```

Comment:
Is there a way to put this project above `matrix-appservice-gitter` on https://matrix.org/bridges/#gitter?

Do we need to make the date newer for `matrix-appservice-gitter` to sort it last?

Reply:
Maybe mark it as `featured: false` so it doesn't appear anymore?

Reply:
Addressing in https://github.com/matrix-org/matrix.org/pull/922